src/test/java/org/apache/commons/lang3/StringUtilsEqualsIndexOfTest.java

```diff
@@ -375,6 +376,18 @@ public void testIndexOfAny_StringCharArray() {
         assertEquals(-1, StringUtils.indexOfAny("ab", 'z'));
     }
 
+
+    @DisplayName("testIndexOfAny_String")
```

Comment:
What's the purpose of having the same for ```@DisplayName``` and test name?

Reply:
Just get extra info https://junit.org/junit5/docs/5.0.3/api/org/junit/jupiter/api/DisplayName.html

Reply:
I don't think we need to crowd the source with DisplayName here since it does not convey more information, give the test method a better name if needed.

Reply:
OK, removed